e2e/tests/vscode.test.ts

```diff
@@ -22,58 +20,62 @@ test.describe('VSCode Tests', () => {
-      vscodeApp.getWindow().getByText('Analysis completed').first()
-    ).toBeVisible({ timeout: 1800000 });
-
-    await vscodeApp.getWindow().screenshot({
-      path: `${SCREENSHOTS_FOLDER}/analysis-finished.png`,
+    await expect(heading).toBeVisible();
+    await vscodeApp.getWindow().waitForTimeout(10000);
+    await window.screenshot({
+      path: `${SCREENSHOTS_FOLDER}/kai-installed-screenshot.png`,
     });
   });
 
-  test('Fix Issue with default (Low) effort', async () => {
-    test.setTimeout(3600000);
-    await vscodeApp.openAnalysisView();
-    const analysisView = await vscodeApp.getAnalysisIframe();
-    await vscodeApp.searchViolation('InventoryEntity');
-    await analysisView
-      .locator('div.pf-v6-c-card__header-toggle')
-      .nth(0)
-      .click();
-    await analysisView.locator('button#get-solution-button').nth(3).click();
-    const resolutionView = await vscodeApp.getResolutionIframe();
-    const fixLocator = resolutionView
-      .locator('button[aria-label="Apply fix"]')
-      .first();
-    await expect(fixLocator).toBeVisible({ timeout: 60000 });
-    await fixLocator.click({ force: true });
+  test('Set Sources and targets', async ({ testRepoData }) => {
+    await vscodeApp.waitDefault();
+    const repoInfo = testRepoData['coolstore'];
+    await vscodeApp.selectSourcesAndTargets(repoInfo.sources, repoInfo.targets);
   });
 
-  test('Fix all issues with default (Low) effort', async () => {
-    test.setTimeout(3600000);
-    await vscodeApp.openAnalysisView();
-    const analysisView = await vscodeApp.getAnalysisIframe();
-    await analysisView
-      .locator('button#get-solution-button')
-      .first()
-      .click({ timeout: 300000 });
-    const resolutionView = await vscodeApp.getResolutionIframe();
-    const fixLocator = resolutionView.locator('button[aria-label="Apply fix"]');
+  test('Set Up Konveyor and Start analyzer', async () => {
+    const window = vscodeApp.getWindow();
+    await vscodeApp.openSetUpKonveyor();
+    await vscodeApp.waitDefault();
+    await window
+      .getByRole('button', { name: 'Configure Generative AI' })
+      .click();
+    await vscodeApp.waitDefault();
+    await window
+      .getByRole('button', { name: 'Configure GenAI model settings file' })
+      .click();
+    await vscodeApp.waitDefault();
 
-    await expect(fixLocator.first()).toBeVisible({ timeout: 3600000 });
-    const fixesNumber = await fixLocator.count();
-    for (let i = 0; i < fixesNumber; i++) {
-      await fixLocator.first().isVisible();
-      await fixLocator.first().click({ force: true });
-    }
+    await window.keyboard.press('Control+a+Delete');
+    await vscodeApp.pasteContent(
+      [
+        'models:',
+        '  AmazonBedrock: &active',
+        '    provider: "ChatBedrock"',
+        '    args:',
+        '      model_id: "meta.llama3-70b-instruct-v1:0"',
+        'active: *active',
+      ].join('\n')
+    );
+    await window.keyboard.press('Control+s');
```

Comment:
This is now on [the vscode class](https://github.com/konveyor/kai-ci/pull/100/files#diff-1fd9549fdef7d4c652f548d28a0c6f1b71b385fec087f877609fa446c3fd598aR287) and its being called in the [global setup](https://github.com/konveyor/kai-ci/pull/100/files#diff-0917a8b5b0606a35c831aaaad8a6fab3e18ff48cdf6eecb31b2decda501f91d3R6), right?